lib/adyen/rest/client.rb

```diff
@@ -68,6 +71,7 @@ def http
       #   of  executing the underlying HTTP request.
       def execute_request(request)
         request.validate!
+        set_endpoint(request.resource)
```

Comment:
I more or l less reset the endpoint logic to what it used to be :   https://github.com/wvanbergen/adyen/pull/148/commits/899dd41d5174805d89091496b9e354beeef77f25 what do you thing?